tests/kafkatest/tests/client/consumer_rolling_upgrade_test.py

```diff
@@ -56,12 +56,7 @@ def _verify_roundrobin_assignment(self, consumer):
         metadata_quorum=[quorum.isolated_kraft],
         use_new_coordinator=[True, False]
     )
-    @matrix(
-        metadata_quorum=quorum.all_kraft,
-        use_new_coordinator=[True],
-        group_protocol=consumer_group.all_group_protocols
```

Comment:
Hey, sorry I'm late to this party, but important concern. Don't we want to keep testing this with the new coordinator and the classic protocol? I would say yes. I would expect we only want to exclude the consumer protocol, so we would want to keep the test with the params it had, but changing `group_protocol=consumer_group.classic`

Reply:
Hey @lianetm. My understanding was that this does test with the new coordinator also after the update, and the default is group_protocol=classic, so this should be fine right? Unless you want to add `combined_kraft`, but not sure if that's an interesting case.

Reply:
You're right, I was only concerned about the new coordinator + classic combination, missed that it's coming from the default, all good then. Thanks!